### PR TITLE
New feat: open any website via computer int

### PR DESCRIPTION
**Pull request by dot-Justin**
`integrations/computer.py`:
   - Deleted unused ComputerParse function
   - New ComputerSite function to search/open any website

`utils/llm_parse.py`:
   - Added ComputerSite
   - Minor reconfig

```
 (\   /)

(◠ + ◠ )
   ►◄
```

### CODE REVIEW

File: utils/llm_parse.py [hunk 1]
```diff
@@ -166,7 +171,7 @@ def LLMParse(user_prompt, last_prompt=None, temperature=0.1, top_p=1):
 def CombinedParse(page, text):
     words = text.split()
     if len(words) <= 1:
-        logging.error("Invalid prompt format.")
+        logging.error("Command did not provide enough parameters.")
```
Comment (dot-Justin) [inline utils/llm_parse.py:174]: Changed this line to make the error more clear

File: utils/llm_parse.py [hunk 2]
```diff
@@ -228,9 +233,12 @@ def CombinedParse(page, text):
                     ComputerVolume(text)
                 else:
                     log_disabled_integration("ComputerVolume")
-            elif recipient in ["run"]:
+            elif recipient == "run":
                 if computerrun_isenabled:
                     ComputerRun(text)
+            elif recipient == "site":
```
Comment (dot-Justin) [inline utils/llm_parse.py:239]: Basic new int config

File: utils/llm_parse.py [hunk 3]
```diff
@@ -241,9 +249,10 @@ def CombinedParse(page, text):
             return
 
         if recipient == "terminate":
-            terminate()
-        else:
-            logging.error("Unknown LAMatHome command or the integration is not enabled.")
+            if lamathometerminate_isenabled:
```
Comment (dot-Justin) [inline utils/llm_parse.py:252]: Added enable/disable for lamathome int

File: utils/llm_parse.py [hunk 4]
```diff
@@ -47,6 +47,9 @@ def LLMParse(user_prompt, last_prompt=None, temperature=0.1, top_p=1):
             Run: "Computer run [search term]"
             Example: "Computer Run command prompt" (Opens command prompt on local computer)
 
+            Site: "Computer site [search term]"
```
Comment (dot-Justin) [inline utils/llm_parse.py:50]: line 50-51, 81, 97, 132 are adding ComputerSite to system prompt

File: integrations/computer.py [hunk 5]
```diff
@@ -8,51 +8,6 @@
 
 computer_isenabled = True
 
-def ComputerParse(browser, title):
```
Comment (dot-Justin) [inline integrations/computer.py:11]: Remove ComputerParse, it's unused

File: integrations/computer.py [hunk 6]
```diff
@@ -259,3 +214,24 @@ def ComputerRun(title):
         logging.info(f"Executed command: {command}")
     except Exception as e:
         logging.error(f"Failed to execute command: {e}")
+
+################################
```
Comment (dot-Justin) [inline integrations/computer.py:218]: New ComputerSite function

**Review by dot-Justin — COMMENTED** [review]
Overall simple change, but the functionality is great.

**Review by pnivek — APPROVED** [review]
LGTM!

**Pull request merged**